mesonbuild/modules/java.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 The Meson development team
```

Comment:
I am also not sure how copyright works here. Do I put my name or leave as is?

Reply:
We don't have a hard and fast rule. When I'm developing on my own time I use `The Meson development team`, when I'm on work time my employer wants me to put `Copyright © 2021 Intel Corporation`, so I do that, and no one has complained

Reply:
I will just leave it as is then! Just did it on my own time.